pro/controllers/users.go

```diff
@@ -114,7 +115,14 @@ func removeUserFromRemoteAccessGW(w http.ResponseWriter, r *http.Request) {
 		}
 		for _, extclient := range extclients {
 			if extclient.OwnerID == user.UserName && remoteGwID == extclient.IngressGatewayID {
-				logic.DeleteExtClient(extclient.Network, extclient.ClientID)
+				err = logic.DeleteExtClient(extclient.Network, extclient.ClientID)
```

Comment:
@yabinma check delete extclient API handler, there are few more cleanup steps involved, for deleting extclient, a wrapper func would be good.